src/api/object_api.js

```diff
@@ -1465,6 +1465,8 @@ module.exports = {
                     properties: {
                     }
                 },
+                err_code: {type: 'string'},
```

Comment:
errors or object infos should be returned inside an array of the reply of delete_multiple_objects functions, not to object_info itself - because there are a lot of functions that return object_info 
please check the reply of delete_multiple_objects https://github.com/noobaa/noobaa-core/blob/master/src/api/object_api.js#L686-L701

Reply:
we should follow the [delete_multiple_objects() reply](https://github.com/noobaa/noobaa-core/blob/master/src/api/object_api.js#L686-L701), 
which means that the array item will be object_info or error, this is the other solution I mentioned in the last comment

Reply:
I've used oneOf to include both object info and the error in the same object of the reply array.
The only other option for same-array reply (I can think of) is explicitly writing all needed fields in object_api reply and in implementation in object_server (which would be unnecessarily verbose).